src/AppInstallerCLICore/ExecutionProgress.cpp

```diff
@@ -133,6 +133,10 @@ namespace AppInstaller::CLI::Execution
     {
         void ProgressVisualizerBase::ApplyStyle(size_t i, size_t max, bool enabled)
         {
+            if (!UseVT() || m_style == VisualStyle::NoVT)

```

Comment:
Can you remove the "m_style == VisualStyle::NoVT" part since it's already checked in UseVT() method? Thanks.

Reply:
Done - I had intended to remove the `case` below as well, as it's now unreachable

Reply:
Thanks. It's ok to leave the NoVT case for completeness but I'm also fine if you removed it.